commands/remove.rb

```diff
@@ -40,12 +40,18 @@ def self.remove(pkg, verbose = CREW_VERBOSE)
     # Use gem to first try to remove gems...
     if pkg.name.start_with?('ruby_')
       @gem_name = pkg.name.sub('ruby_', '').sub('_', '-')
-      if Kernel.system "gem list -i \"^#{@gem_name}\$\""
-        puts "Uninstalling #{@gem_name} before updating. It's ok if this fails.".orange
+      if Kernel.system "gem list -i \"^#{@gem_name}\$\"", %i[out err] => File::NULL
+        puts "Uninstalling #{@gem_name} before removing gem files. It's ok if this fails.".orange
         system "gem uninstall -aIx --abort-on-dependent #{@gem_name}", exception: false
       end
     end
 
+    # Use pip to first try to remove python packages if package uses pip.
+    if pkg.name.start_with?('py3_')
```

Comment:
```suggestion
    if pkg.superclass.eql?('Pip')
```

Reply:
Not doings this because it breaks things...